Content.Server/Administration/Systems/AdminVerbSystem.Antags.cs

```diff
@@ -124,6 +125,22 @@ private void AddAntagVerbs(GetVerbsEvent<Verb> args)
             Message = string.Join(": ", pirateName, Loc.GetString("admin-verb-make-pirate")),
         };
         args.Verbs.Add(pirate);
+        
+        var wizardName = Loc.GetString("admin-verb-text-make-wizard");
+        Verb wizard = new()
+        {
+            Text = wizardName,
+            Category = VerbCategory.Antag,
+            Icon = new SpriteSpecifier.Rsi(new("/Textures/Clothing/Head/Hats/wizardhat.rsi"), "icon"),
+            Act = () =>
+            {
+                //using nukeops rule for wizard, it doesnt do anything anyway
+                _antag.ForceMakeAntag<NukeopsRuleComponent>(targetPlayer, DefaultWizardRule); 
+            },
+            Impact = LogImpact.High,
+            Message = string.Join(": ", wizardName, Loc.GetString("admin-verb-make-wizard")),
+        };
+        args.Verbs.Add(wizard);
```

Comment:
Please add comments so we can easily see edited files
```suggestion
        // Far Horzions start
        var wizardName = Loc.GetString("admin-verb-text-make-wizard");
        Verb wizard = new()
        {
            Text = wizardName,
            Category = VerbCategory.Antag,
            Icon = new SpriteSpecifier.Rsi(new("/Textures/Clothing/Head/Hats/wizardhat.rsi"), "icon"),
            Act = () =>
            {
                //using nukeops rule for wizard, it doesnt do anything anyway
                _antag.ForceMakeAntag<NukeopsRuleComponent>(targetPlayer, DefaultWizardRule); 
            },
            Impact = LogImpact.High,
            Message = string.Join(": ", wizardName, Loc.GetString("admin-verb-make-wizard")),
        };
        args.Verbs.Add(wizard);
        // Far Horizons end
```